extensions.json

```
@@ -1057,8 +1057,8 @@
     {
       "id": "ms-vscode.hexeditor",
       "repository": "https://github.com/microsoft/vscode-hexeditor",
-      "version": "1.3.0",
-      "checkout": "1.3.0"
+      "version": "1.4.0",
+      "checkout": "master"
```

`1.3.0` still seems to be the latest release.

```suggestion
      "version": "1.3.0",
      "checkout": "1.3.0"
```

---

Stopped auto-upgrades: https://github.com/open-vsx/publish-extensions/commit/5467c807519609ffbf56f6483d1bdf2232beb254